src/axom/sidre/tests/sidre_group.cpp

```diff
@@ -2273,12 +2273,6 @@ TEST(sidre_group,import_conduit)
 
   EXPECT_TRUE(ds.getRoot()->importConduitTree(input));
 
-  Group* flds = ds.getRoot()->getGroup("fields");
```

Comment:
Thanks for resolving these warnings @white238.

@agcapps or others -- Is it ok to delete these lines of code? I.e. was there an intention to use these extracted groups, or are these artifacts of some previous way of testing this functionality?